pootle/apps/virtualfolder/views.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.utils.functional import cached_property
+
+from pootle_translationproject.views import TPTranslateView, TPTranslateStoreView
```

Comment:
Shouldn't the order for this be inverted? `TPTranslateStoreView, TPTranslateView`